DemoInfo/DemoParser.cs

```diff
@@ -576,6 +588,21 @@ public bool ParseNextTick()
 						bind.Player = p;
 						RaisePlayerBind(bind);
 					}
+
+					while (p.newWeapons.Count > 0){
+						var weapon = p.newWeapons.Dequeue();
+						if (weapon.Weapon != EquipmentElement.Knife) { 
```

Comment:
Why not also do a pickup-event for the knife?

Reply:
I don't remember if there was a particular reason or if it's just because you can't really drop a knife so it would be redundant.

Would you prefer it to trigger for knives?